front/public/locales/fr/errors.json

```
@@ -33,6 +33,10 @@
       "ConnectionResetByPeer": "Core: réinitialisation de la connexion. Nouvelle tentative.",
       "CoreResponseFormatError": "Core: impossible d'analyser la réponse '{{msg}}'",
       "GenericCoreError": "Core: erreur {{raw_error}}",
+      "Lapin": "Core: file de message: erreur de protocole",
```

I think I don't have the `Lapin` ref :)
It feels weird to read `file de message`, could we add `file d'attente` or we lose some signification with that ?
```suggestion
      "Lapin": "Core: file d'attente de message: erreur de protocole",
```

---

That's the name of the rust library we use to interface with our message queue (RabbitMQ) ;)